docs/how-to/sampling-call-stack.rst

```diff
@@ -0,0 +1,404 @@
+      -G, --gpu-events               Set the GPU hardware counter events to record (ref: `omnitrace-avail -H -c GPU`) (count: unlimited) 
+                                                                  
+      [MISCELLANEOUS OPTIONS]                               
+                                                                  
+      -i, --inlines                  Include inline info in output when available (max: 1, dtype: bool) 
+      --hsa-interrupt [ 0 | 1 ]      Set the value of the HSA_ENABLE_INTERRUPT environment variable.
+                                       ROCm version 5.2 and older have a bug which will cause a deadlock if a sample is taken while waiting for the signal
+                                       that a kernel completed -- which happens when sampling with a real-clock timer. We require this option to be set to
+                                       when --realtime is specified to make users aware that, while this may fix the bug, it can have a negative impact on
+                                       performance.
+                                       Values:
+                                          0     avoid triggering the bug, potentially at the cost of reduced performance
+                                          1     do not modify how ROCm is notified about kernel completion (count: 1, dtype: int)
+
+The general syntax for separating Omnitrace command-line arguments from the 
+following application arguments 
+is consistent with the LLVM style of using a stand-alone double hyphen (``--``). 
+All arguments preceding the double hyphen
+are interpreted as belonging to Omnitrace and all arguments following it 
+are interpreted as the
+application and its arguments. The double hyphen is only necessary when passing 
+command-line arguments to a target
+which also uses hyphens. For example, you can run ``omnitrace-sample ls``, but 
+to run ``ls -la``, use ``omnitrace-sample -- ls -la``.
+
+:doc:`Configuring the Omnitrace runtime options <./configuring-runtime-options>` 
+establishes the precedence of environment variable values over values specified 
+in the configuration files. This enables
+you to configure the Omnitrace runtime to your preferred default behavior 
+in a file such as ``~/.omnitrace.cfg`` and then easily override
+those settings in the command line, for example, ``OMNITRACE_ENABLED=OFF omnitrace-sample -- foo``.
+Similarly, the command-line arguments passed to ``omnitrace-sample`` take precedence 
+over environment variables.
+
+All of the command-line options above correlate to one or more configuration 
+settings, for example, ``--cpu-events`` correlates to the ``OMNITRACE_PAPI_EVENTS`` configuration variable.
+``omnitrace-sample`` processes the arguments and outputs a summary of its configuration 
+before running the target application. 
+
+The following snippets show how ``omnitrace-sample`` runs with various environment updates.
```

Comment:
@dgaliffiAMD This is the part I reformatted based on your feedback. Please let me know if it looks good now.